docs/pages/use-cases/WalletQueries/wallet07.mdx

```diff
@@ -65,7 +166,7 @@ Expected results format
 <Steps>
 ### Select Specific Columns
 
-In order to gather information about staking pools, the query focuses on:
+Similar to our first part, we want to retrieve data for the following columns:
```

Comment:
Maybe on the explanation we can just mentioned that query is similar to the previous one with the addition of the "Where" clause, which then we can explain?

Reply:
@darlisagc updated now, please check, thanks